src/shadcn/ui/checkbox.tsx

```diff
@@ -13,7 +13,7 @@ const Checkbox = React.forwardRef<
   <CheckboxPrimitive.Root
     ref={ref}
     className={cn(
-      "peer h-4 w-4 shrink-0 rounded-sm border border-primary shadow focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:bg-primary data-[state=checked]:text-primary-foreground",
+      "peer h-4 w-4 shrink-0 rounded-sm border border-primary shadow focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:bg-primary data-[state=checked]:text-primary-foreground flex",
```

Comment:
Just some justification issues which can be fixed by adding `justify-center`.
Before:
![Screenshot 2024-08-15 at 10 53 18](https://github.com/user-attachments/assets/09ca4031-a674-4b83-8239-a40b1d953f5b)
After:
![Screenshot 2024-08-15 at 10 53 26](https://github.com/user-attachments/assets/bb6484fd-656f-43c1-9ef4-e5ccbc3b9cd3)

```suggestion
      "peer h-4 w-4 shrink-0 rounded-sm border border-primary shadow focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:bg-primary data-[state=checked]:text-primary-foreground flex justify-center",
```

Reply:
Ah, cheers

Reply:
https://github.com/CQCL/quantinuum-ui/pull/17/commits/250e539df62b286ea3ff373271c684d958e4306b